tests/execute/result/custom.sh

```diff
@@ -16,7 +16,7 @@ rlJournalStart
         rlRun -s "${tmt_command} ${testName} 2>&1 >/dev/null" 1 "Test provides 'results.yaml' file by itself"
         rlAssertGrep "00:11:22 pass /test/custom-results/test/passing" $rlRun_LOG
         rlAssertGrep "00:22:33 fail /test/custom-results/test/failing" $rlRun_LOG
-        rlAssertGrep "00:33:55 pass /test/custom-results \[1/1\]" $rlRun_LOG
+        rlAssertGrep "00:33:55 pass /test/custom-results (on default-0) \[1/1\]" $rlRun_LOG
```

Comment:
Guest info is displayed by default even if executed on a single guest? Similarly for the other test adjustments.

Reply:
This depends on *who* prints the results: `--display-guest` affects the `report` step, but these lines come from the `execute/internal` plugin, which has no notion of this concept of not showing guests. It focuses on its given guest to run tests on and does not look around too much. I'm not sure what can be done here, but with the parallelization, it might be tricky to answer whether there are any parallel threads running for other guests. So, the modified lines are from the `execute` step, not the report plugin.

Reply:
Ah, right, the `execute` step, understood. Perhaps, we could at least ignore the `default-0` name. But, probably, this default name could be sometimes useful as well. Let's keep the polishing part for later if desired.